frontend/uddhar-frontend/src/organization/components/AssignTeam.jsx

```
@@ -68,88 +42,109 @@ const TeamAssignment = () => {
+              />
+              <div className="flex flex-col justify-center items-center mx-4">
+                <button
+                  onClick={addSelectedVolunteers}
+                  className="bg-blue-300 text-black cursor-pointer p-2 rounded mb-2 hover:bg-blue-500 transition duration-200"
+                >
+                  <BadgePlus />
+                </button>
+                <button
+                  onClick={removeSelectedVolunteers}
+                  className="rounded text-red-500 bg-red-100 hover:bg-red-200 cursor-pointer p-2 hover:text-red-700 transition duration-200"
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)